### DISCUSSION TITLE
Algolia Search Engine return "Document exceeds maximum size allowed by Algolia"

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#6136] [Help / Questions] [answered]
Hi, trying to implement Algolia Search Engine on our support website https://support.dcs.film/

I followed the set-up procedure, but when clicked on 'Rebuild Index' WikiJS return an error saying: "Document exceeds maximum size allowed by Algolia". What am I doing wrong?

Thank you in advance for your help.

**Comment by NGPixel** [comment, marked as answer]
Duplicate #3675
